test/integration/components/pythonserver/Dockerfile

```diff
@@ -2,6 +2,7 @@
 FROM python:3.11.4
 EXPOSE 8080
 RUN pip install flask gunicorn
+# Alternative: RUN pip install flask uwsgi
```

Comment:
Added some docs if we want to make a new Python test with the 'other' popular wsgi framework. I tested manually and we capture the events.